### ISSUE TITLE
String like this and "listOfBrokerDealers" should be coming from a property file

### ISSUE THREAD

**Issue by GalateaTom** [galatea-associates/gerrard-sl#30]
https://github.com/galatea-associates/gerrard-sl/blob/d36433222521ec0082408e81f40245b903188508/src/main/java/Initialisation/InitialiseEntities.java#L85

**Comment by Gala-Gerrard** [comment]
Part of the configuration done at the start is that a Map is created with all the paths, imported from a yaml file. The only hard coding are the Keys in the map, which in this case is "listOfCustomers" - is this still too restrictive?

**Comment by GalateaTom** [comment]
Good point.  It's fine to use "listOfBrokerDealers" here then, just please extract it rather than having a magic string
